main.go

```diff
@@ -2,38 +2,25 @@ package main
 
 import (
 	"fmt"
-	"math/rand"
 	"net/http"
-	"time"
-
-	"github.com/VictoriaMetrics/metrics"
 )
 
-var demoCounterTotal = metrics.NewCounter(`demo_counter_total`)
-
 func main() {
-	go func() {
-		for {
-			demoCounterTotal.Inc()
-			time.Sleep(randDurationInRange(time.Millisecond*100, time.Second*2))
-		}
-	}()
+	initMetrics()
+	initAlerting()
 
 	http.Handle("/", http.HandlerFunc(func(rw http.ResponseWriter, _ *http.Request) {
-		rw.Write([]byte(`Welcome! 
+		rw.Write([]byte(`Welcome!
 
-This demo app exposes Prometheus metrics in the standard exposition format. Visit the /metrics endpoint.
-`))
-	}))
+This demo application showcases the capabilities of VictoriaMetrics.
 
-	http.Handle("/metrics", http.HandlerFunc(func(rw http.ResponseWriter, _ *http.Request) {
-		metrics.WritePrometheus(rw, false)
-	}))
+Metrics exposed at /metrics endpoint.
 
-	fmt.Println("Starting server on :9100")
-	http.ListenAndServe(":9100", nil)
-}
+The /alerting/webhook endpoint is used to receive webhooks from the alert manager.
+The /alerting/receivedWebhooks endpoint is used to list the received webhooks.
```

Comment:
NIT: it might worth wrapping URLs in `<a href="...">...` in order to make those clickable enabling quicker path for the user to see the data from these endpoints.

Reply:
updated